pkg/scanfetcher/nexpose.go

```diff
@@ -162,7 +162,7 @@ func scanResourceToCompletedScan(resource resource, start time.Time) (domain.Com
 	// scans are fetched sorted by end time in descending order,
 	// so the first scan resource before the start time signals
 	// that no more scans need to be processed
-	if endTime.Before(start) {
+	if endTime.Before(start) || endTime.Equal(start) {
```

Comment:
nit: "before the start time" should probably be updated to "before or equal to the start time" in the inline code comment above.

Reply:
Fixed